src/daemon/inout_route/chat.rs

```diff
@@ -35,7 +40,7 @@ static CHATS: CtxField<Chats> = |ctx| {
     })
 };
 
-pub fn incoming_chat(ctx: &DaemonContext, neighbor: Fingerprint, msg: String) {
+pub async fn incoming_chat(ctx: &DaemonContext, neighbor: Fingerprint, msg: String) {
```

Comment:
Hmm why was this changed to async? There's no `.await` in it, so this is a bit misleading (since it indicates to the caller that this function could possibly "block")